.github/workflows/dispatch_turbine_go_dependency.yml

```diff
@@ -0,0 +1,48 @@
+name: Remote Dispatch Turbine Go Dependency Responder
+
+on: [repository_dispatch]
+
+jobs:
+  update_turbine_go:
+    if: github.event.action == 'update_dependencies'
```

Comment:
Just curious for my own understanding: how is the event `update_dependencies` defined? Do we have an existing Github action in this repo with the same id?

Reply:
Right! https://github.com/meroxa/turbine-go/pull/52#discussion_r870124687